database/storage.go

```diff
@@ -15,11 +15,44 @@ type Store interface {
 	Rows(string) (*sql.Rows, error)
 }
 
-// Storage ...
+// Storage wraps db connection provided by fromDSN.
 type Storage struct {
 	conn *sql.DB
 }
 
+// CommonStore provides an interface to getTables()
+type CommonStore interface {
+	getTables() ([]string, error)
+}
+
+// IschemaStore provides an interface to access information_schema database.
+// type IschemaStore interface {
+// 	Relationships() ([]string, error)
+// }
+
+// CommonDbs ...
+type CommonDbs struct {
+	iSchemaConn *sql.DB
+	fromDSNconn *sql.DB
+}
+
+// ForeignKeys . . .
+type ForeignKeys struct {
+	fk      string
```

Comment:
Should the fields be private? asking cos `Relationships()` method returns `[]ForeignKeys` so the consumer of this method won't be able to see it's contents if accessing out of database pkg, otherwise it should be a private field too `foreignKeys`